enterprise/server/remote_execution/filecache/filecache_test.go

```diff
@@ -352,6 +358,77 @@ func TestScanWithConcurrentAdd(t *testing.T) {
 	}
 }
 
+func TestFileCacheEvictionAfterSubdirPrefixing(t *testing.T) {
+	ctx := context.Background()
+	fcDir := testfs.MakeTempDir(t)
+
+	baseDir := testfs.MakeTempDir(t)
+
+	{
+		fc, err := filecache.NewFileCache(fcDir, 4096*10, false)
+		if err != nil {
+			t.Fatal(err)
+		}
```

Comment:
nit - might be worth tossing a `WaitForDirectoryScanToComplete` here - idk if the old scanner goroutine will clash with the cache created in the next block below

Reply:
Done.